src/app/cic/controllers/youngScotNationalEntitlementCardDetails.js

```diff
@@ -1,70 +1,8 @@
-const moment = require('moment');
 const BaseController = require("hmpo-form-wizard").Controller;
 const DateControllerMixin = require("hmpo-components").mixins.Date;
-const {APP} = require("../../../lib/config");
 
 const DateController = DateControllerMixin(BaseController);
 
-class YoungScotNationalEntitlementCardDetailsController extends DateController {
+class YoungScotNationalEntitlementCardDetailsController extends DateController {}
```

Comment:
Does it still show the last details entered on the page when going back/going back after Photo Id expired page?